examples/Atlas/Atlas.m

```diff
@@ -349,6 +354,24 @@
 
   properties (SetAccess = protected, GetAccess = public)
     x0
+    hand_right = 0;
+    hand_right_kind = 'none';
+    hand_left = 0;
+    hand_left_kind = 'none';
+    % preconstructing these for efficiency
+    left_full_support
+    left_toe_support
+    right_full_support
+    right_toe_support
+    left_full_right_full_support
+    left_toe_right_full_support
+    left_full_right_toe_support
+    atlas_version = [];
+    external_force = 0; % if nonzero, body id where force is being exerted
+  end
+
+  properties
+    fixed_point_file = fullfile(getDrakePath(), 'examples', 'Atlas', 'data', 'atlas_fp.mat');
```

Comment:
but there is logic in the same file for multiple atlas versions.

yes - only one version in drake, but multiple in drc.

On Apr 20, 2015, at 9:29 AM, Robin Deits <notifications@github.com> wrote:

> In examples/Atlas/Atlas.m:
> 
> > +    hand_left = 0;
> > +    hand_left_kind = 'none';
> > +    % preconstructing these for efficiency
> > +    left_full_support
> > +    left_toe_support
> > +    right_full_support
> > +    right_toe_support
> > +    left_full_right_full_support
> > +    left_toe_right_full_support
> > +    left_full_right_toe_support
> > +    atlas_version = [];
> > +    external_force = 0; % if nonzero, body id where force is being exerted
> > +  end
> > +
> > +  properties
> > +    fixed_point_file = fullfile(getDrakePath(), 'examples', 'Atlas', 'data', 'atlas_fp.mat');
> I don't think we support multiple Atlas versions in Drake, do we? We only have the one urdf
> 
> —
> Reply to this email directly or view it on GitHub.
>